### DISCUSSION TITLE
OBS Audio Configuration

### DISCUSSION THREAD

**Discussion by Chipperchap** [sircharlo/meeting-media-manager#358] [Q&A] [answered]
I am just wanting to get the best out of OBS and am a little confused about the audio configuration (not mentioned in the setup settings [HERE - Initial configuration: OBS Studio](https://sircharlo.github.io/meeting-media-manager/#/present-media))

I have gone into the audio settings and disabled all so that they do not interfere with ZOOM - is that correct?

[<img src="https://simonsheena-gmail.tinytake.com/media/129f002?filename=1653037762519_TinyTake20-05-2022-09-07-18_637886345604912404.png&sub_type=thumbnail_preview&type=attachment&width=800&height=707" title="Powered by TinyTake Screen Capture"/><br><a href="https://www.tinytake.com">Powered by TinyTake Screen Capture</a>](url)

**Comment by sircharlo** [comment, marked as answer]
That's correct, at least in our setup.
Also, I forgot to mention in the docs to [enable stereo audio in Zoom](https://support.zoom.us/hc/en-us/articles/115004830406-Enabling-stereo-audio), in order to transmit the KH "microphone" (which is really the sound mixer's output, including microphones, media and song audio, etc.) in high quality. 

Enable original sound option, high fidelity mode and stereo audio, like so:
![image](https://user-images.githubusercontent.com/1275666/169517838-568fe409-2bbc-4dec-bd80-8b0b36594b5c.png)

Then, once in the Zoom meeting, enable original sound in the top left corner:
![image](https://user-images.githubusercontent.com/1275666/169518053-0ea7c9bf-b3ff-4990-b2f6-a9353bcde7d9.png)

Of course, the "microphone" chosen in Zoom would have to be the KH "microphone" (which is really the sound mixer's output, including microphones, media and song audio, etc.) corresponding to the mixer feed.
